src/app/pages/details/details.hbs

```diff
@@ -0,0 +1,172 @@
+<style scoped>
+    @import url(/app/pages/details/details.css);
+    @import url(https://fonts.googleapis.com/css?family=Lato:400,900);
+</style>
+<section class="floating-menu">
```

Comment:
`section` tags are defined on MDN as the following:
> The HTML &lt;section&gt; element represents a generic section of a document, i.e., a thematic grouping of content, typically with a heading. Each &lt;section&gt; should be identified, typically by including a heading (&lt;h1&gt;-&lt;h6&gt; element) as a child of the &lt;section&gt; element.

There are a large number of section elements throughout this template that appear to be inappropriate.  They should probably just be `div`s.